README.md

```diff
@@ -1 +1,17 @@
-# hw01
\ No newline at end of file
+# hw01
+
+## **About Me**
```

Comment:
The use of the header here is a good idea. This isn't really a critique, but you could include another header later in the file as well, maybe when you transition into talking about nonacademic interests.